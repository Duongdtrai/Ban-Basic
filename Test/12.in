4
10
95280 49523 95480 94490 41795 37608 44962 34010 51146 18997 
4
94987 68704 83320 74575 
2
62366 29815 
8
63663 18014 64348 54794 93058 87512 89794 66969 
