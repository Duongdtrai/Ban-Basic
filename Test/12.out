241140804520
48730368560
0
377317742796
